### PR TITLE
Milestone1: Create Rail API app

### PR DESCRIPTION
**Pull request by Jmagero**
- [x] Create a new Rails API app called 'hello-rails-back-end'.
- [x] Initialize your project with Git.
- [x] Make sure that your project has a Postgres database set up.
- [x] Create a table for storing your messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action an rails route).
- [x] Create a new React app called 'hello-react-front-end'
- [x] Initialize your project with Git.
- [x] Create the App component with react-router.
- [x] Create the Greeting component that will display a greeting. Set it up as a route in your App component.
- [x] Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
- [x] Display the random greeting in your Greeting component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,76 @@
...
+
+### Running locally
+
+1. Start server:
+
+``` bash
+rails server
+```
+
+2. Go to the endpoint `http://localhost:3000/api/v1/greetings` to access the API.
```
Comment (DeliceLydia) [inline README.md]: - [ ] Modify this API to `http://localhost:3000/api/v1/random-greeting`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,76 @@
...
+```
+3. Install dependencies:
+``` bash
+bundle install
+```
+4. Setup database:
+``` bash
+rails db:create
+rails db:migrate
+```
```
Comment (DeliceLydia) [inline README.md]: - [ ] Add `rails db:seed` to seed the database

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Jmagero, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**